src/challengePaths/2021/YouRobot.ts

```diff
@@ -0,0 +1,113 @@
+import {
+  myRobotEnergy,
+  myRobotScraps,
+  round,
+  runChoice,
+  Slot,
+  Stat,
+  visitUrl,
+} from "kolmafia";
+import { have } from "../../lib.js";
+import { get } from "../../property.js";
+import { $path, $skill, $slot, $stat } from "../../template-string.js";
+
+export const path = $path`You, Robot`;
+
+type Upgrade = "robot_shirt" | "robot_energy" | "robot_potions";
+
+export class YouRobot {
+  static scrap(): number {
+    return myRobotScraps();
+  }
+
+  static doScavenge(): void {
```

Comment:
Functions should return a meaningful value if they can, and this one absolutely can.